examples/node/lib/rest/api/v2010/account.ts

```diff
@@ -83,7 +83,7 @@ export interface AccountListInstanceCreateOptions {
  */
 export interface AccountListInstanceEachOptions {
   dateCreated?: Date;
-  dateTest?: Date;
```

Comment:
I understand the purpose of the change here, but I've never seen properties in quotes before. Is this standard in Typescript?